### ISSUE TITLE
Layout/EmptyLineBetweenDefs not useful for single line class definitions?

### ISSUE THREAD

**Issue by Aqualon** [rubocop-hq/rubocop#9088]
We define our own error classes like this and get it flagged by `Layout/EmptyLineBetweenDefs`.

```
module ModelErrors
  class Gone < StandardError; end
  class Invalid < StandardError; end
  class NotFound < StandardError; end
  class Retry < StandardError; end
end
```
--------

## Expected behavior

The cop should not report `Use empty lines between class definitions` for single line classes.

## Actual behavior

The cop reports `Use empty lines between class definitions` but I don't see how this would improve the code.

## Steps to reproduce the problem

`rubocop --only Layout/EmptyLineBetweenDefs` on the example.

## RuboCop version

```
$ [bundle exec] rubocop -V
1.4.1 (using Parser 2.7.2.0, rubocop-ast 1.1.1, running on ruby 2.7.2 x86_64-darwin19)
```

**Comment by bbatsov** [comment]
There's an option controlling this - `AllowAdjacentOneLineDefs`.

**Comment by Aqualon** [comment]
Thanks, this worked.
